<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Preset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="presetBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="DBRadio">
        <property name="text">
         <string>Delany-Bazley</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="DBMRadio">
        <property name="text">
         <string>Delany-Bazley-Miki</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="JCARadio">
        <property name="text">
         <string>JCA</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Thickness (mm)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="thicknessBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Porosity</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="phiBox">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.950000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Tortuosity</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="tortuosityBox">
        <property name="value">
         <double>1.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Flow resistivity (k)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="sigmaBox">
        <property name="value">
         <double>12.400000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Visc Len (μm)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="vLengthBox">
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Therm Len (μm)</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="tLengthBox">
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>thicknessBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phiBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tortuosityBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sigmaBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vLengthBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tLengthBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePlot()</slot>
 </slots>
</ui>
